packages/contentstack-export/src/app.js

```diff
@@ -15,7 +15,7 @@ exports.initial = async (config) => {
     exports.getConfig = () => {
       return config;
     };
-    const APIClient = stack.Client(config);
+    const APIClient = await managementSDKClient(config);
```

Comment:
@netrajpatel We are not using any async functionality in the managementSDKClient; can we remove async and use it as a simple function instead of waiting for it?

Reply:
@antonyagustine We have to because the latest SDK version has introduced a promised call for a retry connection event in case of 401, to handle that in the future we have to make use of await or promise to get the managementSDKClient.
So by doing this, we will be making sure to reduce as many SDK connections and share the same connection instance whenever required, as well as those will be used as a promise function.

Reply:
Let me know your thoughts.

Reply:
Sure we can keep it if we have a plan for it in future.